static/library/slider_component.ts

```diff
@@ -249,47 +319,98 @@ export class DatacommonsSliderComponent extends LitElement {
       `;
     }
 
+    const startDate = this._dates[0];
+    const endDate = this._dates[this._dates.length - 1];
+    // Normalized slider value as a percent between 0 and 100
+    const normalizedSliderValue =
+      this._showLatestAvailableData || this._dates.length <= 1
+        ? 100
+        : (this._value / (this._dates.length - 1)) * 100;
+
+    const dateText = this.getDateText();
+    const endDateText = this.getEndDateText();
+    const lastDateIndex = this._dates.length - 1;
+    const isHighestCoverageDate = dateText === this._highestCoverageDate;
+
+    // Text to show under range slider button.
+    // Shows asterisk if showing latest available data or date of highest
+    // coverage
+    const sliderLabelText = `${
+      this._showLatestAvailableData ? endDateText : dateText
```

Comment:
this was slightly confusing because of the different cases all being handled in one line, maybe can do something like:

```
// <note the nuance in the text being different for showLatestAvailabeData>
let sliderLabelText = this._showLatestAvailableData ? endDataText : dateText
// Add asterisk if ...
if (this._showLatestAvailableData || isHighestCoverageDate) {
    sliderLabelText += "*"
}

Reply:
Updated